pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   compiler_unsupported: "^1.22.0"
   crypto: ">=1.0.0 <3.0.0"
   glob: "^1.0.0"
+  html: "^0.13.0"
```

Comment:
Note that once https://github.com/dart-lang/pub/issues/1544 is resolved this can be removed.